browser/ui/brave_browser_command_controller_browsertest.cc

```diff
@@ -155,6 +155,7 @@ IN_PROC_BROWSER_TEST_F(BraveBrowserCommandControllerTest,
   EXPECT_TRUE(command_controller->IsCommandEnabled(
                   IDC_SHOW_BRAVE_WEBCOMPAT_REPORTER));
 
+#if BUILDFLAG(ENABLE_TOR)
```

Comment:
I think some other tests code below (ex, `EXPECT_TRUE(command_controller->IsCommandEnabled(IDC_SHOW_BRAVE_REWARDS));` should be run even if tor is disabled.
But, it will not be run with this new `#if`.

Reply:
How can we create a tor window when `ENABLE_TOR` is false?
You can try building this PR with enable_tor=false and you can see why we need to avoid calling `brave::NewOffTheRecordWindowTor`

Reply:
Ah.. I see! below `command_controller` comes from `tor_browser`. NVM my comment!